src/main/java/com/google/api/codegen/discovery/transformer/go/GoSampleTypeNameConverter.java

```diff
@@ -72,13 +72,9 @@ public TypeName getRequestTypeName(String apiTypeName, TypeInfo typeInfo) {
 
   @Override
   public TypeName getTypeName(TypeInfo typeInfo) {
-    if (typeInfo.isMessage()) {
-      String localName = GoSampleNamer.getServicePackageName(packagePrefix);
-      return new TypeName(packagePrefix + ";;;", localName + "." + typeInfo.message().typeName());
```

Comment:
The package prefix bit never really did anything, it can be used to make sure the type's package is imported, but we've already imported that package in the transformer